spec/presenters/sufia/file_set_presenter_spec.rb

```diff
@@ -4,7 +4,17 @@
   let(:solr_document) { SolrDocument.new(file.to_solr) }
   let(:ability) { double "Ability" }
   let(:presenter) { described_class.new(solr_document, ability) }
-  let(:file) { build(:file_set).tap { |f| f.apply_depositor_metadata(user) } }
+  let(:file) do
+    build(:file_set).tap do |f|
+      f.id = '123abc'
+      f.apply_depositor_metadata(user)
```

Comment:
Instead of this block, just pass args to `build(:file_set, id: '123abc', user: user)`